### ISSUE TITLE
evaluation and dataset analysis

### ISSUE THREAD

**Issue by chrishokamp** [qe-team/marmot#21]
we need utilities for comparing datasets and analyzing the statistical significance of results. 

Questions that we should be able to answer
- which tokens in the test set are OOV with respect to the training data?
- what is the statistical significance of our performance on a dev set -- i.e. are we doing better than random, or the most frequent class baseline?
